packages/schema/bind/src/bindings/assemblyscript/wasm/index.ts

```diff
@@ -136,12 +136,15 @@ export const generateBinding: GenerateBindingFn = (
 
   // Generate module type folders
   if (abi.moduleType) {
+    const moduleType = abi.envType
+      ? { ...abi.moduleType, hasEnv: true }
```

Comment:
The proper way to do this would be to add another transform, see here: https://github.com/polywrap/toolchain/blob/65d7416b5cb78aa51a9267903afb66c5c6e7a975/packages/schema/bind/src/bindings/assemblyscript/wasm/index.ts#L187-L198

Reply:
Yeah for sure!